cdap-app-fabric/src/test/java/io/cdap/cdap/internal/app/services/http/AppFabricTestBase.java

```diff
@@ -225,7 +226,8 @@ public abstract class AppFabricTestBase {
 
   @BeforeClass
   public static void beforeClass() throws Throwable {
-    initializeAndStartServices(createBasicCConf());
+    cConfiguration = createBasicCConf();
```

Comment:
you actually can just set the flag to false here, feel most places do not need it to be true